test/context/toJSON.js

```diff
@@ -1,7 +1,7 @@
 
 'use strict';
 
-const context = require('../context');
+const context = require('.');
```

Comment:
hmm, as far as i can tell `'.'` is just shorthand for `'./'` in node's directory system... but may be wrong.